### PR TITLE
added ohm to the dictionary

### PR DESCRIPTION
**Pull request by Xraycat**
Closed a previous PR with ohm because i had taken an inverted ohm symbol

### CODE REVIEW

**Comment by Xraycat** [comment]
added ohm to the dictionary

**Comment by DenverCoder1** [comment]
Looks like a good addition, but please check that the spacing is correct as explained in issue #5, otherwise the dictionary won't import.

There should be a tab before and after the symbol as seen on previous lines. You can copy the symbol from there.

Also keep in mind, you can edit the existing PR by making more commits on the same branch (in your case, https://github.com/Xraycat/LaTeX-Gboard-Dictionary/tree/patch-3). There's no need to close and reopen it  :smile: 

Let me know if you have any questions! I'm happy to help.

File: dictionary.txt [hunk 1]
```diff
@@ -907,3 +907,4 @@ _rp	₎
 \QED	∎	
 \male	♂	
 \female	♀	
+\ohm Ω
```
Comment (DenverCoder1) [inline dictionary.txt:910]:
```suggestion
\ohm	Ω	
```

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for contributing! :tada:

**Pull request merged**